lib/internal/test_runner/reporter/tap.js

```diff
@@ -46,8 +46,14 @@ async function * tapReporter(source) {
         yield `${indent(data.nesting)}# Subtest: ${tapEscape(data.name)}\n`;
         break;
       case 'test:stderr':
-      case 'test:stdout':
-      case 'test:diagnostic':
+      case 'test:stdout': {
+        const lines = StringPrototypeSplit(data.message, kLineBreakRegExp);
```

Comment:
i think this might not be safe, and you need:
```suggestion
        const lines = RegExpPrototypeSymbolSplit(kLineBreakRegExp, data.message);
```